### PR TITLE
Http server boilerplate

### PR DESCRIPTION
**Pull request by marmoure**
Adds another workspace to the prosemirror-lwdita project.

 This PR, adds the server boilerplate code to start development.
 
 How to test this PR:
 * install the new dependencies by running `yarn install`.
 * start the http server by building and starting it.
 ```shell
 $ cd packages/prosemirror-lwdita-backend
 $ yarn run build
 $ yarn run start
 ```
* visit http://localhost:3000/ and check for the server response.

### CODE REVIEW

File: packages/prosemirror-lwdita-backend/README.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# prosemirror-lwdita-backend
+API module for prosemirror lwdita
+
+This is a standalone module with it's own http server.
```
Comment (plutonik-a) [inline packages/prosemirror-lwdita-backend/README.md:4]: @marmoure Can you please re-check this entire paragraph. It contains some typos.

File: packages/prosemirror-lwdita-backend/src/server.ts [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import express from 'express';
```
Comment (plutonik-a) [inline packages/prosemirror-lwdita-backend/src/server.ts:1]: @marmoure Could you please provide a basic API test that we can extend later once we define the routes?

File: packages/prosemirror-lwdita-backend/README.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+```shell
+$ yarn run build
+```
+
+Start the server:
+```shell
+$ yarn run start
+```
+
+# API documenting
```
Comment (plutonik-a) [inline packages/prosemirror-lwdita-backend/README.md:26]: @marmoure Typo & grammar, please fix.

**Review by plutonik-a — CHANGES_REQUESTED** [review]

File: packages/prosemirror-lwdita-backend/src/server.ts (hunk 2 above)
Comment (marmoure) [reply packages/prosemirror-lwdita-backend/src/server.ts:1]: @plutonik-a That would be time costly, as we have to decide on which framework needed for testing and how exactly should we test the API.
can we leave it until we finish the happy case MVP?

File: packages/prosemirror-lwdita-backend/README.md (hunk 3 above)
Comment (marmoure) [reply packages/prosemirror-lwdita-backend/README.md]: fixed in fe0ad59982f00b312827a75cb369cd835786c518

File: packages/prosemirror-lwdita-backend/README.md (hunk 1 above)
Comment (marmoure) [reply packages/prosemirror-lwdita-backend/README.md]: fixed in fe0ad59982f00b312827a75cb369cd835786c518

File: packages/prosemirror-lwdita-backend/src/server.ts (hunk 2 above)
Comment (plutonik-a) [reply packages/prosemirror-lwdita-backend/src/server.ts:1]: We will add the test at a later point then and add the open task in our backlog.

**Pull request merged**